src/OSPSuite.Infrastructure.Import/Core/DataFormat/AbstractColumnsDataFormat.cs

```diff
@@ -173,11 +173,12 @@ protected virtual void ExtractGeneralParameters(List<string> keys, IUnformattedD
       public IEnumerable<ParsedDataSet> Parse(IUnformattedData data,
          IReadOnlyList<ColumnInfo> columnInfos)
       {
-         var groupByParams =
+         var groupingCriteria =
             Parameters
-               .Where(p => p is GroupByDataFormatParameter || p is MetaDataFormatParameter)
+               .Where(p => p is GroupByDataFormatParameter || p is MetaDataFormatParameter && p.ColumnName != null)//this last check makes sure that a MetaDataFormatParameter with manual input (eg molecule)
```

Comment:
put the comment above the line.. I don't want to have to scroll to read the command that makes the come clearer

Reply:
done